monkey/monkey_island/cc/services/aws_service.py

```diff
@@ -15,59 +15,83 @@
 logger = logging.getLogger(__name__)
 
 
-def filter_instance_data_from_aws_response(response):
+class AWSService:
+    def __init__(self, aws_instance: AWSInstance):
+        """
+        :param aws_instance: An AWSInstance object representing the AWS instance that the Island is
+                             running on
+        """
+        self._aws_instance = aws_instance
+
+    def island_is_running_on_aws(self) -> bool:
```

Comment:
`is_island_running_on_aws`

Reply:
We talked about not using hungarian notation for functions that return boolean. See https://guardicore.atlassian.net/wiki/spaces/RES/pages/2800353435/Infection+Monkey+Development+Guidelines+DRAFT#Functions%2FMethods-returning-booleans